spec/services/stripe_model_callbacks/invoice/updated_service_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+describe StripeModelCallbacks::Invoice::UpdatedService do
+  subject(:updated_service) { described_class.execute!(event: stripe_event) }
+
+  let(:stripe_event) { Stripe::Event.construct_from(event_values) }
+  let(:event_payload) { File.read("spec/fixtures/stripe/invoice/created.json") }
+  let(:event_values) do
+    {
+      id: "evt_1GiGbaGJAmuZ9ZZZbKpF4hGj",
+      type: event_type,
+      data: JSON.parse(event_payload, symbolize_names: true)
+    }
+  end
+  let(:stripe_id) { "in_1GxxxxxxZ9ZZZxxXLFQx" }
+
+  shared_examples "do not create new invoice" do
+    it "does not create new stripe invoice" do
+      expect { updated_service }.not_to change(StripeInvoice.all, :size)
```

Comment:
`size` can be cached
```suggestion
      expect { updated_service }.not_to change(StripeInvoice.all, :count)
```